test/view_test.go

```diff
@@ -500,3 +500,67 @@ func TestUseArangoSearchView(t *testing.T) {
 		}
 	}
 }
+
+// TestGetArangoSearchView creates an arangosearch view and then gets it again.
+func TestArangoSearchViewProperties35(t *testing.T) {
+	ctx := context.Background()
+	c := createClientFromEnv(t, true)
+	skipBelowVersion(c, "3.5", t)
```

Comment:
Does this actually terminate the test code here in case the test is skipped? And if so, how does this work? In other places we have an if and an expiicit return.